tests/test_gradual_padding.py

```
@@ -0,0 +1,102 @@
+    """Test cases for gradual_padding behaviour."""
+
+    @pytest.mark.core
+    def test_gradual_padding_28(self):
+        """Test gradual padding of 28 days extends segments into flat regions."""
+        df = pt.load_data('series_synthetic')
+        results = pt.detect_trends(
+            df,
+            date_col='date',
+            value_col='gradual',
+            plot=False,
+            method_params=dict(gradual_padding=28)
```

Ruff (0.15.21)

[warning] Unnecessary `dict()` call (rewrite as a literal)

Rewrite as a literal

(C408)